### PR TITLE
Add python 3.11 to ci

### PR DESCRIPTION
**Pull request by simlay**

### CODE REVIEW

**Comment by simlay** [comment]
Looks like the publish flow is broken. I think https://github.com/pypa/cibuildwheel/pull/1300 might be the fix?

**Comment by joerick** [comment]
heyo. happened to see the trackback link on https://github.com/pypa/cibuildwheel/pull/1300 . Unlikely to be that - check your build identifiers, they look wrong to me.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest] # TODO: add windows-2019,
-        python-version: ["37", "38", "39", "310"]
+        python-version: ["37", "38", "39", "310", "3.11"]
```
Comment (henryiii) [inline .github/workflows/publish.yml:17]:
```suggestion
        python-version: ["37", "38", "39", "310", "311"]
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
...
     env:
       CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* pp* *-win32"
```
Comment (henryiii) [inline .github/workflows/publish.yml:19]:
```suggestion
      CIBW_SKIP: "cp36-* pp* *-win32"
```

**Comment by simlay** [comment]
> heyo. happened to see the trackback link on [pypa/cibuildwheel#1300](https://github.com/pypa/cibuildwheel/pull/1300) . Unlikely to be that - check your build identifiers, they look wrong to me.

Ah, yes you're right. Thanks a bunch!

**Pull request merged**